### DISCUSSION TITLE
Ability to add an external image element

### DISCUSSION THREAD

**Discussion by vijaysridhara** [mcmonkeyprojects/SwarmUI#341] [Ideas]
I want to add a specific image element to an existing image, in the "Edit image" option as a new layer. this could help a lot in generating a perfectly composed image as expected. Please consider adding this feature. The new image element could be a png/jpg file from the Computer folders .

**Comment by mcmonkey4eva** [comment]
just drag an image onto the editor and it should immediate add as a layer
